docs/react/guides/migrating-to-react-query-5.md

```diff
@@ -26,3 +26,39 @@ if you still need to remove a query, you can use `queryClient.removeQueries({que
 ### The minimum required TypeScript version is now 4.7
 
 Mainly because an important fix was shipped around type inference. Please see this [TypeScript issue](https://github.com/microsoft/TypeScript/issues/43371) for more information.
+
+### Removed `keepPreviousData` in favor of `placeholderData` identity function
+
+We have removed the `keepPreviousData` option and `isPreviousData` flag as they were doing mostly the same thing as `placeholderData` and `isPlaceholderData` flag.
+
+To achieve the same functionality as `keepPreviousData`, we have added previous query `data` as an argument to `placeholderData` function.
+Therefore you just need to provide an identity function to `placeholderData`
+
+```diff
+useQuery({
+  queryKey,
+  queryFn,
+- keepPreviousData: true,
++ placeholderData: (previousData) => previousData
+});
```

Comment:
let's also add in the diff here that `isPreviousData` is now `isPlaceholderData`

Reply:
✅